<launch>
    <node name="lsm10_v2" pkg="lsm10_v2" type="lsm10_v2" output="screen" ns="LeftLidar">
    <param name="scan_topic" value="scan"/>         	        #设置激光数据topic名称
    <param name="frame_id" value="laser_link"/>     	        #激光坐标
    <param name="serial_port" value="/dev/ttyUSB0"/>        	#雷达连接的串口
    <param name="angle_disable_min" value="0.0"/>		#雷达裁剪角度开始值
    <param name="angle_disable_max" value="0.0"/>		#雷达裁剪角度结束值
    <param name="min_range" value="0.0"/>			#雷达接收距离最小值
    <param name="max_range" value="100.0"/>			#雷达接收距离最大值
    <param name="versions" value="1"/>			#雷达协议选择 1 : 1.0.1协议/ 2 : 1.3.2协议
    </node>
    
    <node name="lsm10_v2" pkg="lsm10_v2" type="lsm10_v2" output="screen" ns="RightLidar">
    <param name="scan_topic" value="scan"/>         	        #设置激光数据topic名称
    <param name="frame_id" value="laser_link"/>     	        #激光坐标
    <param name="serial_port" value="/dev/ttyUSB1"/>        	#雷达连接的串口
    <param name="angle_disable_min" value="0.0"/>		#雷达裁剪角度开始值
    <param name="angle_disable_max" value="0.0"/>		#雷达裁剪角度结束值
    <param name="min_range" value="0.0"/>			#雷达接收距离最小值
    <param name="max_range" value="100.0"/>			#雷达接收距离最大值
    <param name="versions" value="1"/>			#雷达协议选择 1 : 1.0.1协议/ 2 : 1.3.2协议
    </node>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lsm10_v2)/rviz/lslidar.rviz" output="screen"/>      #雷达点云配置

</launch>

